### PR TITLE
Fix muxer help to promote sdk when absent

### PR DESCRIPTION
**Pull request by schellap**

### CODE REVIEW

**Comment by schellap** [comment]
Cc @gkhanna79

**Comment by schellap** [comment]
Multiple inputs about the message being confusing to users.

File: src/corehost/cli/fxr/fx_muxer.cpp [hunk 1]
```diff
@@ -580,9 +587,13 @@ int muxer_usage()
...
     trace::println();
     trace::println(_X("If you are debugging the Shared Framework Host, set 'COREHOST_TRACE' to '1' in your environment."));
-    trace::println();
-    trace::println(_X("To get started on developing applications for .NET Core, install .NET SDK from:"));
-    trace::println(_X("  %s"), DOTNET_CORE_URL);
+
+    if (include_sdk_link)
+    {
+        trace::println();
+        trace::println(_X("To get started on developing applications for .NET Core, install .NET SDK from:"));
```
Comment (gkhanna79) [inline src/corehost/cli/fxr/fx_muxer.cpp]: I would suggest to rephrase this as "...install the SDK from: "

**Comment by gkhanna79** [comment]
@piotrpMSFT should chime in on this.

File: src/corehost/cli/fxr/fx_muxer.cpp (hunk 1 above)
Comment (schellap) [inline src/corehost/cli/fxr/fx_muxer.cpp]: See https://github.com/dotnet/core-setup/pull/201/files

Cc @hickford

**Comment by piotrpMSFT** [comment]
If the SDK is present we should guide folks towards using it. Perhaps recommending 'dotnet --help'?

/Cc @blackdwarf

**Comment by blackdwarf** [comment]
If we have SDK on the machine, I would say that the message above should not appear, that is, something else should be said. For instance, if the SDK is present:
> Use dotnet --help to get help with the SDK

If SDK is not installed/present:

> To get started on developing applications for .NET Core, install .NET SDK from: https://aka.ms/core-sdk

In this way, the user doesn't get confused by the message since we are aware of the context.

**Comment by hickford** [comment]
Good idea. I was confused why the app nagged me to install the SDK when I'd just installed it.
